src/platform/datapath_winkernel.c

```diff
@@ -2923,6 +2973,20 @@ SocketSend(
         *(PINT)WSA_CMSG_DATA(CMsg) = SendData->ECN;
     }
 
+    if (Binding->Datapath->Features & CXPLAT_DATAPATH_FEATURE_SEND_DSCP) {
```

Comment:
The lines above are already trying to set `IP_ECN`. Should we be setting both that and this?

Reply:
I was wondering about that. I was thinking about only setting IP_ECN when `CXPLAT_DATAPATH_FEATURE_SEND_DSCP` is not supported and only setting DSCP+ECN when DSCP is supported